agent/command/papertrail_trace_test.go

```diff
@@ -213,6 +213,184 @@ func TestPapertrailTrace(t *testing.T) {
 	}
 }
 
+func TestPapertrailGetFiles(t *testing.T) {
+	type testFile struct {
+		path    string
+		content string
+		want    papertrailTraceFile
+	}
+
+	cases := map[string]struct {
+		files    []testFile
+		patterns []string
+	}{
+		"simple": {
+			files: []testFile{
+				{
+					path:    "test.tar.gz",
+					content: "test1",
+					want: papertrailTraceFile{
+						filename: "test.tar.gz",
+						sha256:   "1b4f0e9851971998e732078544c96b36c3d01cedf7caa332359d6f1d83567014",
+					},
+				},
+			},
+			patterns: []string{"test.tar.gz"},
+		},
+		"nested-simple": {
+			files: []testFile{
+				{
+					path:    "parent/test.tar.gz",
+					content: "test1",
+					want: papertrailTraceFile{
+						filename: "test.tar.gz",
+						sha256:   "1b4f0e9851971998e732078544c96b36c3d01cedf7caa332359d6f1d83567014",
+					},
+				},
+			},
+			patterns: []string{"parent/test.tar.gz"},
+		},
+		"wildcard": {
+			files: []testFile{
+				{
+					path:    "test1.tar.gz",
+					content: "foo",
+					want: papertrailTraceFile{
+						filename: "test1.tar.gz",
+						sha256:   "2c26b46b68ffc68ff99b453c1d30413413422d706483bfa0f98a5e886266e7ae",
+					},
+				},
+				{
+					path:    "test2.tar.gz",
+					content: "bar",
+					want: papertrailTraceFile{
+						filename: "test2.tar.gz",
+						sha256:   "fcde2b2edba56bf408601fb721fe9b5c338d10ee429ea04fae5511b68fbf8fb9",
+					},
+				},
+			},
+			patterns: []string{"*.tar.gz"},
+		},
+		"wildcard-parent": {
+			files: []testFile{
+				{
+					path:    "parent/test1.tar.gz",
+					content: "foo",
+					want: papertrailTraceFile{
+						filename: "test1.tar.gz",
+						sha256:   "2c26b46b68ffc68ff99b453c1d30413413422d706483bfa0f98a5e886266e7ae",
+					},
+				},
+				{
+					path:    "parent/test2.tar.gz",
+					content: "bar",
+					want: papertrailTraceFile{
+						filename: "test2.tar.gz",
+						sha256:   "fcde2b2edba56bf408601fb721fe9b5c338d10ee429ea04fae5511b68fbf8fb9",
+					},
+				},
+			},
+			patterns: []string{"parent/*.tar.gz"},
+		},
+		"multiple-wildcard-parent": {
+			files: []testFile{
+				{
+					path:    "test-1/parent/test1.tar.gz",
+					content: "foo",
+					want: papertrailTraceFile{
+						filename: "test1.tar.gz",
+						sha256:   "2c26b46b68ffc68ff99b453c1d30413413422d706483bfa0f98a5e886266e7ae",
+					},
+				},
+				{
+					path:    "test-2/parent/test2.tar.gz",
+					content: "bar",
+					want: papertrailTraceFile{
+						filename: "test2.tar.gz",
+						sha256:   "fcde2b2edba56bf408601fb721fe9b5c338d10ee429ea04fae5511b68fbf8fb9",
+					},
+				},
+			},
+			patterns: []string{"*/parent/*.tar.gz"},
+		},
+	}
+
+	for name, tc := range cases {
+		name, tc := name, tc
+		t.Run(name, func(t *testing.T) {
+			tempdir, err := os.MkdirTemp(os.TempDir(), "papertrail-get-files-test-*")
+			if err != nil {
+				t.Fatalf("make temp dir: %s", err)
```

Comment:
It's more conventional in Evergreen to use testify for assert/require rather than use the built-in `testing` package. Can we change this test to use testify?

If we want to unblock this for users, I'm also okay with making a separate ticket to clean up the tests in this file to follow the Evergreen conventions.